tests/src/Integration/Action/SystemMailToUsersOfRoleTest.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\rules\Integration\Action\SystemMailToUsersOfRoleTest.
+ */
+
+namespace Drupal\Tests\rules\Integration\Action;
+
+use Drupal\Core\Entity\EntityManagerInterface;
+use Drupal\Tests\rules\Integration\RulesEntityIntegrationTestBase;
+use Drupal\Component\Utility\SafeMarkup;
+use Drupal\Core\Language\LanguageInterface;
+use Drupal\Core\Mail\MailManagerInterface;
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\user\RoleInterface;
+use Drupal\user\UserInterface;
+use Drupal\user\UserStorageInterface;
+use Psr\Log\LoggerInterface;
+
+
+/**
+ * @coversDefaultClass \Drupal\rules\Plugin\RulesAction\SystemMailToUsersOfRole
+ * @group rules_actions
+ */
+class SystemMailToUsersOfRoleTest extends RulesEntityIntegrationTestBase {
+
+  /**
+   * @var \Psr\Log\LoggerInterface
+   */
+  protected $logger;
+
+  /**
+   * @var \Drupal\Core\Mail\MailManagerInterface
+   */
+  protected $mailManager;
+
+  /**
+   * @var \Drupal\user\UserStorage
+   */
+  protected $userStorage;
+
+  /**
+   * @var \Drupal\user\Entity\Role[]
+   */
+  protected $roles;
+
+  /**
+   * @var \Drupal\user\Entity\User[]
+   */
+  protected $users;
+
+  /**
+   * The action to be tested.
+   *
+   * @var \Drupal\rules\Plugin\RulesAction\SystemMailToUsersOfRole
+   */
+  protected $action;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+    $this->enableModule('user');
+    // @todo why is that line needed? Please add a comment.
+    $this->namespaces[] = '';
```

Comment:
So is this line still needed? Did you try without that line? What was the error in that case?